src/Elm/Print.hs

```diff
@@ -206,11 +213,152 @@ elmEnumUniverse t@ElmType{..} = vsep
 arrow :: Doc ann
 arrow = "->"
 
+----------------------------------------------------------------------------
+-- Encode
+----------------------------------------------------------------------------
+
+{- | Returns the encoder for the given type.
+
+
+TODO
+ +-------------------+------------------+-----------------+---------------------+
+ |    Haskell Type   |     Eml Type     |     Encoder     |       JSON          |
+ +===================+==================+=================+=====================+
+ |   'Int'           |      'Int'       | standard encoder |                    |
```

Comment:
I like the idea of having this table! It can be filled later if we will write big documentation for this package.